target/linux/ramips/image/mt76x8.mk

```diff
@@ -213,6 +213,20 @@ define Device/tl-wr841n-v13
 endef
 TARGET_DEVICES += tl-wr841n-v13
 
+define Device/tl-wr841n-v14
+  $(Device/tplink)
+  DTS := TL-WR841NV14
+  IMAGE_SIZE := 3968k
+  DEVICE_TITLE := TP-Link TL-WR841N v14
+  TPLINK_FLASHLAYOUT := 4Mmtk
+  TPLINK_HWID := 0x08410014
+  TPLINK_HWREV := 0x1
+  TPLINK_HWREVADD := 0x14
+  TPLINK_HVERSION := 3
+  IMAGE/tftp-recovery.bin := pad-extra 64k | $$(IMAGE/factory.bin)
+endef
+TARGET_DEVICES += tl-wr841n-v14
```

Comment:
And `TARGET_DEVICES += tplink_tl-wr841n-v14`

Reply:
There should be only `TARGET_DEVICES += tplink_tl-wr841n-v14` and not both `TARGET_DEVICES += tplink_tl-wr841n-v14` and `TARGET_DEVICES += tl-wr841n-v14`, this won't even compile.

Reply:
Oh. I don't know how I could have overlooked this. Sorry, fixed.